python/requirements.txt

```diff
@@ -0,0 +1,7 @@
+Flask
+azure-ai-textanalytics
+Werkzeug
+
+#Flask==2.1.1
+#azure-ai-textanalytics==5.2.0b2
+#Werkzeug==2.2.2
```

Comment:
Assume you're going to keep one or the other before merging?

Reply:
yanked the comments. They were vestiges of a past life :)